source/sync/configure/pause-or-terminate-sync.txt

```diff
@@ -105,17 +214,147 @@ all clients.
+            #. Select the :guilabel:`mongodb-atlas CLUSTER`.
+            
+            The URL in your browser's toolbar should now resemble:
+
+            ``https://realm.mongodb.com/groups/$GROUP_ID/apps/$APP_ID/services/$SERVICE_ID/config``
+
+         .. step:: Get the Sync Service Configuration
+
+            Send a GET request to the 
+            `adminGetServiceConfig 
+            <https://www.mongodb.com/docs/atlas/app-services/admin/api/v3/#tag/services/operation/adminGetServiceConfig>`__ 
+            endpoint to get the Device Sync service configuration. This returns
+            a Service configuration object similar to:
+
+            .. code-block:: json
+
+                {
+                  "clusterId": "<MY-CLUSTER-ID>",
+                  "clusterName": "Cluster0",
+                  "clusterType": "atlas",
+                  "flexible_sync": {
+                    "state": "disabled",
+                    "database_name": "todo",
+                    "queryable_fields_version": 1,
+                    "permissions": {
+                      "rules": {},
+                      "defaultRoles": []
+                    },
+                    "client_max_offline_days": 30,
+                    "is_recovery_mode_disabled": false
+                  },
+                  "groupName": "<MY-GROUP-NAME>",
+                  "orgName": "<MY-ORG-NAME>",
+                  "readPreference": "primary",
+                  "wireProtocolEnabled": false
+                }
+
+            For details about the Sync configuration returned by this endpoint,
+            refer to :ref:`sync-configuration-reference`.
+
+         .. step:: Edit the Sync Service Configuration
+
+            Remove just the ``flexible_sync`` part of the configuration to
+            a new JSON object. This configuration contains a ``state`` field 
+            that reflects the current state of the Sync protocol for the 
+            application. Set this value to ``enabled``. This should
+            now resemble a JSON blob similar to:
+
+            .. code-block:: json
+               :emphasize-lines: 3
+
+                {
+                  "flexible_sync": {
+                    "state": "enabled",
+                    "database_name": "todo",
+                    "queryable_fields_version": 1,
+                    "permissions": {
+                      "rules": {},
+                      "defaultRoles": []
+                    },
+                    "client_max_offline_days": 30,
+                    "is_recovery_mode_disabled": false
+                  }
+                }
+
+         .. step:: Update the Configuration on Atlas
+
+            Send the updated Sync Service configuration as a payload in a 
+            PATCH request to the `adminUpdateServiceConfig 
+            <https://www.mongodb.com/docs/atlas/app-services/admin/api/v3/#tag/services/operation/adminUpdateServiceConfig>`__ 
+            endpoint.
+
+   .. tab:: Realm SDKs
```

Comment:
I'm confused why there's an SDK tab in Terminate Sync section if the linked procedures are to _pause_ and resume a session, not _terminate_ it?

Reply:
Ignore me 😅 thought this was in the other section